data/fixtures/scopes/textual/boundedParagraph2.scope

```diff
@@ -0,0 +1,211 @@
+aaa
+bbb {
+    ccc
+    ddd
+
+    eee
+    fff
+
+    ggg
+    hhh
+} iii
+jjj
+
+kkk
+---
+
+[#1 Content] =
+[#1 Domain] = 0:0-3:7
+  >---
+0| aaa
+1| bbb {
+2|     ccc
+3|     ddd
+   -------<
+
+[#1 Removal] = 0:0-5:0
+  >---
+0| aaa
+1| bbb {
+2|     ccc
+3|     ddd
+4|
+5|     eee
+   <
+
+[#1 Trailing delimiter: Content] = 4:0-4:0
+  ><
+4|
+[#1 Trailing delimiter: Removal] = 4:0-5:0
+  >
+4|
+5|     eee
+   <
+
+[#1 Insertion delimiter] = "\n\n"
+
+
+[#2 Content] =
+[#2 Domain] = 2:4-3:7
+      >---
+2|     ccc
+3|     ddd
+   -------<
+
+[#2 Removal] = 2:0-5:0
+  >-------
+2|     ccc
+3|     ddd
+4|
+5|     eee
+   <
+
+[#2 Trailing delimiter: Content] = 4:0-4:0
+  ><
+4|
+[#2 Trailing delimiter: Removal] = 4:0-5:0
+  >
+4|
+5|     eee
+   <
+
+[#2 Insertion delimiter] = "\n\n"
+
+
+[#3 Content] = 5:4-6:7
+      >---
+5|     eee
+6|     fff
+   -------<
+
+[#3 Removal] = 5:0-8:0
+  >-------
+5|     eee
+6|     fff
+7|
+8|     ggg
+   <
+
+[#3 Leading delimiter: Content] = 4:0-4:0
+  ><
+4|
+[#3 Leading delimiter: Removal] = 4:0-5:0
+  >
+4|
+5|     eee
+   <
+
+[#3 Trailing delimiter: Content] = 7:0-7:0
+  ><
+7|
+[#3 Trailing delimiter: Removal] = 7:0-8:0
+  >
+7|
+8|     ggg
+   <
+
+[#3 Domain] = 5:0-6:7
+  >-------
+5|     eee
+6|     fff
+   -------<
+
+[#3 Insertion delimiter] = "\n\n"
+
+
+[#4 Content] =
+[#4 Domain] = 8:0-9:7
+  >-------
+8|     ggg
+9|     hhh
+   -------<
```

Comment:
this content range is wrong, but the domain seems ~good. Tempted to have end go to start of `}` but I don't feel strongly